content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst

```diff
@@ -1,161 +1,240 @@
+<inventory/shipping_receiving/configure-delivery-method>` in Odoo, create or navigate to a quotation
+by going to :menuselection:`Sales app --> Orders --> Quotations`.
 
-Delivery
---------
+.. _inventory/shipping_receiving/third-party-so:
 
-The delivery created from the sale order will take the shipping
-information from it. You can change the carrier if you want to.
+Sales order
+-----------
 
-If you create a delivery transfer from the inventory module, you can add
-the third party shipper in the additional info tab.
+To assign a third party shipping carrier and get an estimated cost of shipping, begin by going to
+:menuselection:`Sales app --> Orders --> Quotations`. Create a select an existing quotation, and add
+the cost of shipping through a third-party carrier to a quotation by clicking the :guilabel:`Add
+Shipping` button in the bottom-right corner.
 
-.. image:: third_party_shipper/third_party_shipper01.png
+.. image:: ../advanced_operations_shipping/invoicing/add-shipping.png
    :align: center
+   :alt: Show the "Add shipping" button at the bottom of a quotation.
+
+In the resulting pop-up window, select the intended carrier from the :guilabel:`Shipping Method`
+drop-down menu. The :guilabel:`Cost` field is automatically filled based on the :ref:`product weight
+<inventory/shipping_receiving/configure-weight>` and distance between the warehouse's :ref:`source
+address <inventory/shipping_receiving/configure-source-address>` and the customer address.
 
-The weight is computed based on the products weights. The shipping cost
-will be computed once the transfer is validated.
+.. _inventory/shipping_receiving/third-party-rate:
 
-Click on **Validate** to receive the tracking number. The **Carrier Tracking
-ref** field will automatically be filled. Click on the **Tracking** button
-to check your delivery on the provider website.
+Click :guilabel:`Get Rate` in the :guilabel:`Add a shipping method` pop-up window to get the
+estimated cost through the shipping connector. Then, click the :guilabel:`Add` button to add the
+delivery charge to the :abbr:`SO (Sales Order)` or invoice.
 
-.. image:: third_party_shipper/third_party_shipper08.png
+.. image:: ../advanced_operations_shipping/invoicing/add-a-shipping-method.png
    :align: center
+   :alt: Show the "Get rate" button in the "Add a shipping method" pop-up.
 
-The label to stick on your package is available in the history
-underneath :
+.. seealso::
+   :doc:`Charge customers for shipping after product delivery
+   <../advanced_operations_shipping/invoicing>`
 
-.. image:: third_party_shipper/third_party_shipper04.png
-   :align: center
+.. _inventory/shipping_receiving/third-party-do:
+
+Delivery order
+--------------
+
+For users making shipments without installing the *Sales* app, assign the shipping carrier to the
+delivery order, by first going to the :menuselection:`Inventory` app. Select the :guilabel:`Delivery
+Orders` operation type, and choose the desired delivery order.
```

Comment:
do you think it would be helpful to mention that the user should choose a DO that isn't already marked as 'Done' -- or do you think that is redundant info?